### DISCUSSION TITLE
Check if session exists for a provided folder

### DISCUSSION THREAD

**Discussion by augustocdias** [rmagatti/auto-session#119] [Q&A]
Is there a way to check if a session exists for a given folder? I basically want to use this plugin: https://github.com/nickgerace/gfold

and when I open a directory I want to trigger the session save, close it and if a session for the new cwd exists, I want to restore it. Reading the code I could figure out how to do all the steps but check if there's a session for the given `cwd`...

**Comment by rmagatti** [comment]
There's currently not a function that checks if a session exists for the current cwd. It could be fairly easily added though, I'd be happy to review a PR for it!
